### DISCUSSION TITLE
jarvis reduziert die internen Sicherungen nach einigen Tagen

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1538] [Q&A]
Kann es sein, dass jarvis nach einigen Tagen die internen Sicherungen von 8 auf 3 automatisch reduziert?
In der Sicherung (BackitUp) sind immer wieder größere Schwankungen in der Größe zu erkennen. 
![image](https://user-images.githubusercontent.com/84818934/150641977-8dd503ef-d276-463d-b47c-8ad94b06ac70.png)
An den internen Sicherungen verändert man da ja nichts selbst?

**Comment by Zefau** [comment]
Nein. Es werden immer 25 Versionen gespeichert (also je 25 von Devices, Layout, Widgets und Settings).

**Comment by mcuiobroker** [comment]
Dann wird es anscheinend mit jedem Update reduziert?

**Comment by Zefau** [comment]
Jo, wird bei jedem Update gelöscht meine ich (aufgrund der ioBroker Infrastruktur)

**Comment by mcuiobroker** [comment]
Das ist nicht gut. Dann musst du vielleicht _old-Dateien erstellen, also vorher umbenennen?
Nur weil man ein Update macht, hat man plötzlich keine interne Sicherung mehr.
So hat man noch die Möglichkeit durch umbenennen sich die alten Sicherungen zurückzuholen.
Eigentlich musst du die Dateien doch nur neu schreiben, wenn sie nicht vorhanden sind?
